### PR TITLE
[Op] Adding `div` op

### LINKED ISSUE #283: Implement div op end to end

### PR DESCRIPTION
**Pull request by mtopalovicTT**
[Op] Adding `div` op

resolves #283

### CODE REVIEW

File: include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td [hunk 1]
```diff
@@ -96,6 +96,15 @@ def TTKernel_AddOp : TTKernel_Op<"add"> {
     let arguments = (ins I32:$dst_index);
 }
 
+def TTKernel_DivOp : TTKernel_Op<"div"> {
+    let summary = "Divide operation";
+    let description = [{
+      Divide operation
+    }];
+
+    let arguments = (ins I32:$dst_index);
```
Comment (nsmithtt) [inline include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td:105]: We shouldn't need to update TTKernel dialect, the AddOp above even is incorrect.  We are still bringing up this dialect.  The rest looks good though!

**Review by nsmithtt — APPROVED** [review]
Please remove the updates to the TTKernel dialect, but otherwise looks good!

File: include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td (hunk 1 above)
Comment (mtopalovicTT) [reply include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td:105]: Sure, i will remove addop to. Other elwise ops are supported?

Comment (nsmithtt) [reply include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td:105]: Leave the add op, I think it's being used by a test, but we're going to rework it.

**Review by jnie-TT — APPROVED** [review]

**Review by svuckovicTT — APPROVED** [review]

**Review by sdjordjevicTT — APPROVED** [review]

**Pull request merged**